docs/attributes-registry/db.md

```diff
@@ -170,23 +170,23 @@ This group defines attributes for Azure Cosmos DB.
-| `ExecuteJavaScript` | execute_javascript | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `Head`              | head               | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `HeadFeed`          | head_feed          | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `Invalid`           | invalid            | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `Patch`             | patch              | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `Query`             | query              | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `QueryPlan`         | query_plan         | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `Read`              | read               | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `ReadFeed`          | read_feed          | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `Replace`           | replace            | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `Upsert`            | upsert             | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| Value                | Description        | Stability                                                        |
+| -------------------- | ------------------ | ---------------------------------------------------------------- |
+| `batch`              | batch              | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `create`             | create             | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `delete`             | delete             | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `execute`            | execute            | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `execute_javascript` | execute_javascript | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `head`               | head               | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `head_feed`          | head_feed          | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `invalid`            | invalid            | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `patch`              | patch              | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `query`              | query              | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `query_plan`         | query_plan         | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `read`               | read               | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `read_feed`          | read_feed          | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
probably worth adding "change_feed" in addition to read_feed - because we have to different APIs in the SDK - for normal read feed vs. changefeed

Reply:
IN java there is also a dedicated read_all API (to read all documents of a container or all documents of a logical partition) - we can probably group it under read_feed - but read_many should definitely be its own operation name. SO, my proposal would be to add read_all and read_many and change_feed instead of read_feed